cmd/helm/repo_update.go

```diff
@@ -53,12 +62,15 @@ func newRepoUpdateCmd(out io.Writer) *cobra.Command {
 		update: updateCharts,
 	}
 	cmd := &cobra.Command{
-		Use:     "update",
+		Use:     "update [repo_name]",
```

Comment:
:s/repo_name/REPO_NAME
for consistency with commands like `helm get`, `helm install` etc.

Reply:
done